### PR TITLE
Add non-scrolling behaviour for attachments on Welcome screen

### PR DESCRIPTION
**Pull request by igorkravchenko**
Disable scrolling for attachments on Message Center Welcome screen in favour of scrolling the whole screen.

MOB-2099

### CODE REVIEW

**Review by yurii-glia — APPROVED** [review]

File: GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListViewModel.swift [hunk 1]
```diff
@@ -108,10 +109,15 @@ extension SecureConversations {
...
     var maxUnscrollableViews: Int {
-        if environment.uiApplication.preferredContentSizeCategory() <= .accessibilityMedium {
-            return .maxUnscrollableViewsOnDefaultContentSizeCategory
-        } else {
-            return .maxUnscrollableViewsOnLargeContentSizeCategory
+        switch environment.scrollingBehaviour {
+        case .nonScrolling:
+            return .max
+        case let .scrolling(application):
+            if application.preferredContentSizeCategory() <= .accessibilityMedium {
```
Comment (gersonnoboa) [inline GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListViewModel.swift:116]: Would be nice to know the logic behind this. At least for me it's not obvious.

Comment (igorkravchenko) [reply GliaWidgets/SecureConversations/FileUploadListView/SecureConversations.FileUploadListViewModel.swift:128]: Added documentation and comments 04c21f51df42d6893d9a52162669ad35a94f559b.

**Review by gersonnoboa — APPROVED** [review]

**Comment by igorkravchenko** [comment]
!squash

**Comment by igorkravchenko** [comment]
!merge

**Pull request merged**